src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -818,6 +819,20 @@ public String getVisibilityDropdownLabel(int qnNumber) {
                                                          + " .visibility-options-dropdown button")).getText().trim();
     }
 
+    public boolean isStaticQnNumberVisible(int qnNumber) {
+        WebElement discardChangesButton =
```

Comment:
The element you are referencing is not `discardChanges` button. Lets rename the variable in both the testing functions to what the element actually is.